pynio/service.py

```diff
@@ -114,6 +114,13 @@ def create_block(self, name, type, config=None):
             blk.save()
         return blk
 
+    def hasblock(self, block):
```

Comment:
I don't think this is a good method for a Service. Instead we can simplify the service by just providing a `blocks` property that returns a list (or maybe a dict) of all the blocks it contains. Then a user can easily do what he wants with that list, including a simple has_block:
block in service.blocks